web_widget_one2many_product_picker/readme/USAGE.rst

```diff
@@ -0,0 +1,120 @@
+Widget options:
```

Comment:
This should be CONFIG.rst, not USAGE.rst, and remember to put the GIF in USAGE.rst